seatunnel-connectors-v2/connector-iceberg/src/main/java/org/apache/seatunnel/connectors/seatunnel/iceberg/source/IcebergSource.java

```diff
@@ -105,9 +105,15 @@ private SeaTunnelRowType loadSeaTunnelRowType(Schema tableSchema, Config pluginC
                         columnDataTypes.toArray(new SeaTunnelDataType[0]));
 
         CheckResult checkResult =
-                CheckConfigUtil.checkAllExists(pluginConfig, CommonConfig.KEY_FIELDS.key());
+                CheckConfigUtil.checkAtLeastOneExists(
+                        pluginConfig, CommonConfig.KEY_FIELDS.key(), SeaTunnelSchema.SCHEMA.key());
+
         if (checkResult.isSuccess()) {
-            SeaTunnelSchema configSchema = SeaTunnelSchema.buildWithConfig(pluginConfig);
+            Config config =
+                    pluginConfig.hasPath(SeaTunnelSchema.SCHEMA.key())
+                            ? pluginConfig.getConfig(SeaTunnelSchema.SCHEMA.key())
+                            : pluginConfig;
```

Comment:
Do you mean `pluginConfig.getConfig(CommonConfig.KEY_FIELDS.key())`?

Reply:
Yes, thank you very much! I will fix this.

Reply:
I mean if user use  `schema`  in config file , we use `pluginConfig.getConfig(SeaTunnelSchema.SCHEMA.key())`.  And to support some old tasks which only use `fields` we just use orginal config.

Reply:
> Do you mean `pluginConfig.getConfig(CommonConfig.KEY_FIELDS.key())`?

I mean if user use  `schema`  in config file , we use `pluginConfig.getConfig(SeaTunnelSchema.SCHEMA.key())`.  And to support some old tasks which only use `fields` we just use orginal config.

Reply:
Got it.